ddtrace/contrib/pyramid/patch.py

```diff
@@ -37,3 +39,20 @@ def traced_init(wrapped, instance, args, kwargs):
 
     wrapped(*args, **kwargs)
     trace_pyramid(instance)
+
+def insert_tween_if_needed(settings):
+    if 'pyramid.tweens' not in settings:
```

Comment:
```python
tweens_list = settings.get('pyramid.tweens')
if not tweens_list:
    return
```

Covers both if the list is empty or if the key is not defined.

Reply:
The way it's done now, I strip the string from all spaces characters before checking if it is empty to avoid the corner case where the users would pass "  \n \t" as a "pyramid.tweens" that would be considered as an empty string by pyramid.

Reply:
can't you:
```python
tweens = settings.get('pyramid.tweens')
if not tweens and not tweens.strip() and DD_TWEEN_NAME in tweens:
    return
# the rest of your code
```